src/pages/iou/IOUCurrencySelection.js

```diff
@@ -59,6 +59,7 @@ class IOUCurrencySelection extends Component {
     constructor(props) {
         super(props);
 
+        this.selectedCurrencyCode = lodashGet(props.route, 'params.currency', props.iou.selectedCurrencyCode);
```

Comment:
Looks like this affects how we render the component so it should be a state variable

Reply:
Good catch, thanks!

Reply:
Yes it will affect the render, but it never changes, so I was questioning whether it's important to put it in a state or not.

For now, I have put it in a state as requested.